docs/source/install/rhel6.rst

```diff
@@ -62,8 +75,8 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
     sudo chkconfig mongod on
     sudo chkconfig rabbitmq-server on
 
-    # Install and configure postgres 9.4
-    sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-1.noarch.rpm
+    # Install and configure postgres 9.4. Based on OS type install the ``redhat`` one or ``centos`` one.
+    sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-${os}-94-9.4-1.noarch.rpm
```

Comment:
People mostly copy-paste. Let's assume they are doing RHEL, and have another commented line for CentOS. 
Something like:  
```
# Install and configure postgres 9.4. Based on OS type install the ``redhat`` one or ``centos`` one.
# RHEL:
sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat-94-9.4-1.noarch.rpm
# CentOS:
sudo yum -y localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos-94-9.4-1.noarch.rpm
```